code/API_definitions/one-time-password-sms.yaml

```
@@ -179,6 +179,7 @@ components:
     PhoneNumber:
       description: A phone number belonging to the user. 'E164 with +' format
```

We internally usually use: "^\+\d{1,15}"
But if we want '+'  to be optional we should update the description as well.

```suggestion
      description: A phone number belonging to the user. E.164 format (starting with country code). Optionally prefixed with '+'.
```